apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -121,6 +121,28 @@ export function InstructorCourseAdminInstances({
                 </tbody>
               </table>
             </div>
+            ${courseInstances.length === 0
```

Comment:
What do you think about conditionally rendering the whole table? That is, if there aren't any course instances, it may make sense to hide the table and its headers, as the headers don't do anything in the empty state.

Reply:
That sounds good -- I modified the page to hide the table headers if there are no course instances:

<img width="901" alt="image" src="https://github.com/user-attachments/assets/9e7e9d17-a6ca-4bc2-a597-69b244012640">